src/cmap/commands.ts

```diff
@@ -837,13 +838,18 @@ export class BinMsg {
     const promoteValues = options.promoteValues ?? this.opts.promoteValues;
     const promoteBuffers = options.promoteBuffers ?? this.opts.promoteBuffers;
     const bsonRegExp = options.bsonRegExp ?? this.opts.bsonRegExp;
+    if (options.validation && Object.keys(options.validation.utf8)[0] !== 'writeErrors') {
+      throw new Error('Can only toggle validation settings for writeErrors key');
```

Comment:
I think we should probably use one of our custom error classes here, like `MongoInvalidArgumentError` in this case.